tests/test_utils.tsx

```diff
@@ -28,6 +28,20 @@ import { NodeProcess } from "./fixtures_islands/NodeProcess.tsx";
 import { FreshAttrs } from "./fixtures_islands/FreshAttrs.tsx";
 import { OptOutPartialLink } from "./fixtures_islands/OptOutPartialLink.tsx";
 
+const browser = await launch({
+  args: [
+    "--window-size=1280,7201",
+    ...((Deno.env.get("CI") && Deno.build.os === "linux")
+      ? ["--no-sandbox"]
```

Comment:
https://github.com/lino-levan/astral?tab=readme-ov-file#no-usable-sandbox-with-user-namespace-cloning-enabled

I would recommend removing app armor instead of enabling --no-sandbox

Reply:
What is the advantage? Genuine question.

Reply:
Or do you think that's the issue here?

Reply:
The biggest reason is it cleans up the codebase. I've heard of really obscure bugs not being caught in CI because of `--no-sandbox`, but I've never seen one myself to take that with a grain of salt.